symbolic/am_or.m

```diff
@@ -1,6 +1,11 @@
 function fun = am_or(a,b)
-% syms x y
-% f = symfun(sym('cw_or(x,y)'),[x y]);
-% fun = f(a,b);
+% am_or is the amici implementation of the symbolic or function
+%
+% Parameters:
+%  a: first input parameter @type sym
+%  b: second input parameter @type sym
+%
+% Return values:
+%  fun: logical value, negative for false, positive for true
 fun = am_max(a,b);
```

Comment:
that mapping is implicitely done in AMICI at runtime. In CVODES events are defined as roots with odd multiplicity. In SBML events always fire when triggers transition from false to true.

The advantage of the continuous mapping as that math is a little bit more rigorous and the "distance" to true/false can be used for regularisation of event observables that did not happen during simulation